packages/terminology/src/operations/conceptMapImport.ts

```diff
@@ -0,0 +1,118 @@
+import { FhirRequest, FhirResponse } from "@medplum/fhir-router";
+import { ConceptMap } from "@medplum/fhirtypes";
+import { normalizeOperationOutcome } from "@medplum/core";
+import { v4 as uuidv4 } from "uuid";
+
+import { getSqliteClient } from "../sqlite";
+
+function generateReverseConceptMaps(conceptMap: ConceptMap): ConceptMap[] {
+  const reverseMaps: ConceptMap[] = [];
+  const sourceSystem = conceptMap.group?.[0]?.source;
+  const targetSystem = conceptMap.group?.[0]?.target;
+  const sourceCode = conceptMap.group?.[0]?.element?.[0]?.code;
+  const targets = conceptMap.group?.[0]?.element?.[0]?.target;
+
+  if (!sourceSystem || !targetSystem || !sourceCode || !targets) {
+    return [];
+  }
+
+  for (const target of targets) {
+    const reverseMap: ConceptMap = {
+      resourceType: "ConceptMap",
+      status: "active",
+      group: [
+        {
+          source: targetSystem,
+          target: sourceSystem,
+          element: [
+            {
+              code: target.code,
+              target: [
+                {
+                  code: sourceCode,
+                  equivalence: targets.length === 1 ? "equivalent" : "wider",
+                },
+              ],
+            },
+          ],
+        },
+      ],
+    };
+    reverseMaps.push(reverseMap);
+  }
+
+  return reverseMaps;
+}
+
+export async function conceptMapImportHandler(
```

Comment:
there are a lot of codes with multiple entries. We need to do upserts here to support that.